code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -501,43 +488,21 @@ components:
                       format: int32
                       minimum: 1
                       maximum: 65535
-                      description: Port number exposed by the component. OP may generate a dynamic port towards the UCs 
-                                          corresponding to this internal port and forward the client traffic from dynamic port to containerPort.
-                    interfaceName:
-                      type: string
-                      description:  Interface Name. Required only if application has to be attatched to a network other then default.
+                      description: Port number exposed by the component. OP may generate a dynamic port towards the component instance which forwards external traffic to the component port.
                     visibilityType:
-                      description: Defines whether the interface is exposed to outer world or not i.e., external, or internal.
-                                          If this is set to "external", then it is  exposed to external applications otherwise it is exposed 
-                                          internally to edge application components within edge cloud. When exposed to external world, 
-                                          an external dynamic port is assigned for UC traffic and mapped to the internal containerPort
+                      description: Defines whether the interface is exposed to outer world or not i.e., external, or internal.If this is set to "external", then it is  exposed to external applications otherwise it is exposed internally to edge application components within edge cloud. When exposed to external world, an external dynamic port is assigned for UC traffic and mapped to the extPort
                       type: string
                       enum:
                         - VISIBILITY_EXTERNAL
                         - VISIBILITY_INTERNAL
                 minItems: 1                
-              numOfInstances:
-                type: integer
-                format: int32
-                description: Number of component instances to be launched. 
-              restartPolicy:
-                type: string
-                enum:
-                  - RESTART_POLICY_ALWAYS
-                  - RESTART_POLICY_NEVER
-                description: How the platform shall handle component failure                        
+
       required:
         - name
         - virtType
-        - fileName
         - repository
         - componentSpec
-    CpuArchType:
-          type: string
-          enum:
-            - x86_64
-            - arm_64
-          description: CPU Instruction Set Architecture (ISA) E.g., Intel, Arm etc. 
```

Comment:
Not sure if we should remove this. We sometimes see container images build for specific arch. I would like to see what others experience is here.

Reply:
Clear, Thanks!